.vscode/launch.json

```
@@ -0,0 +1,37 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "pwa-node",
+            "request": "launch",
+            "name": "Launch Program",
+            "skipFiles": [
+                "<node_internals>/**"
+            ],
+            "program": "${workspaceFolder}/@here/harp-examples/webpack.config.js",
+            "preLaunchTask": "tsc: build - tsconfig.json",
+            "outFiles": [
+                "${workspaceFolder}/dist/node_modules/**/*.js"
+            ]
+        },
+        {
+            "type": "chrome",
+            "request": "launch",
+            "name": "Karma Tests",
+            "sourceMaps": true,
+            "webRoot": "${workspaceRoot}/test",
+            "url": "http://localhost:9876/debug.html",
```

yarn test-debug launches a browser window, and then this launches another one. We could instead attach to the browser already launched by test-debug command.
```suggestion
            "request": "attach",
            "name": "Karma Tests",
            "sourceMaps": true,
            "webRoot": "${workspaceRoot}/test",
            "address": "localhost",
            "port": 9333,
```

---

done